Jenkinsfile

```diff
@@ -231,39 +231,50 @@ pipeline {
             ])
           }
         }
-        stage('engine-IT-tomcat-9-postgresql-142') {
+        stage('engine-IT-tomcat-9-postgresql-170') {
```

Comment:
❓  Do we still need stages with `tomcat-9`?

Reply:
Which also might mean that we keep only the CI but remove the tomcat 9 webapp module. 🤔 I will check